tracks/crystal/exercises/pangram/mentoring.md

```diff
@@ -0,0 +1,56 @@
+### Algorithm
+
+There are lots of ways to solve this one. Methods that operate on the whole collection are preferred to iterative methods.
+
+```crystal
```

Comment:
It would be helpful to add a line above or below each solution so mentors can _skim_ for the correct match. You can use heading level 4, or regular text.